content/desktop/contributing-and-collaborating-using-github-desktop/committing-and-reviewing-changes-to-your-project.md

```diff
@@ -35,48 +35,51 @@ As you make changes to files in your text editor and save them locally, you will
 
-If one file contains multiple changes, but you only want *some* of those changes to be included in a commit, you can create a partial commit. The rest of your changes will remain intact, so that you can make additional modifications and commits. This allows you to make separate, meaningful commits, such as keeping line break changes in a commit separate from code or prose changes.
+If one file contains multiple changes, but you only want some of those changes to be included in a commit, you can create a partial commit. The rest of your changes will remain intact, so that you can make additional modifications and commits. This allows you to make separate, meaningful commits, such as keeping line break changes in a commit separate from code or prose changes.
 
-When you review the diff of the file, the lines that will be included in the commit are highlighted in blue. To exclude the change, click the changed line so the blue disappears.
+{% note %}
+
+**Note:** Split diff displays are currently in beta and subject to change.
+
+{% endnote %}
 
-![Unselected lines in a file](/assets/images/help/desktop/partial-commit.png)
+1. In the top right corner, use {% octicon "gear" aria-label="The Gear icon" %} to select **Unified** or **Split** to review your changes.
+  ![Gear icon with unified and split diffs](/assets/images/help/desktop/gear-diff-select.png)
+2. The lines that will be included in the commit are highlighted in blue. To exclude the change, click one or more changed lines so the blue disappears.
+  ![Unselected lines in a file](/assets/images/help/desktop/partial-commit.png)
 
-#### Discarding changes
+### 3. Discarding changes
+If you have uncommitted changes that you don't want, you can remove the changes by discarding them. You can discard all uncommitted changes in one file, a range of files, or discard all changes in all files since the last commit.
 
-You can discard all the uncommitted changes in one file, a range of files, or discard all changes in all files since the last commit.
+Discarded changes are saved in a dated file in the Trash and you can recover them until the Trash is emptied.
 
-{% mac %}
+#### Discarding changes in one or more files
 
 {% data reusables.desktop.select-discard-files %}
 {% data reusables.desktop.click-discard-files %}
   ![Discard Changes option in context menu](/assets/images/help/desktop/discard-changes-mac.png)
 {% data reusables.desktop.confirm-discard-files %}
   ![Discard Changes button in the confirmation dialog](/assets/images/help/desktop/discard-changes-confirm-mac.png)
 
-{% tip %}
+#### Discarding changes in one or more lines
+You can discard one or more added lines in a single file before you commit them.
 
-**Tip:** The changes you discarded are saved in a dated file in the Trash and you can recover them until the Trash is emptied.
-
-{% endtip %}
+{% note %}
 
-{% endmac %}
+**Note:** Discarding single lines is disabled in a group of changes that adds and removes lines.
```

Comment:
I think this one should probably stay a note? When I was trying out the feature this caused me a lot of confusion.  If you have both additions and deletions in the same chunk of changes *discard changes* is greyed out, but I could not for the life of me figure out why. I think this could end up being a big driver of users coming to the docs.

Reply:
>Use callouts sparingly for high-value information - do not include general information, permissions, or prerequisites in callouts.

>Notes provide useful information or reminders for the user, but the information is not required to follow. Notes may not be relevant or necessary to every user.

IMO, this is both general information and necessary for every user to understand how the feature works. That being said, I'll trust you to make the call. 😊